src/nvim/options.lua

```diff
@@ -1480,6 +1480,28 @@ return {
       type = 'string',
       varname = 'p_cot',
     },
+    {
+      abbreviation = 'cpp',
+      cb = 'did_set_completepopup',
+      defaults = { if_true = '' },
+      desc = [=[
+        This option is used to set the completion window and info floating
+        preview window. The option is a comma-separated list of values:
+        `align`, `border`, `title`, `titlepos`, `footer`, `footerpos`.
+        see |nvim_open_win()|.
+        When the "align" value is "item" then the info floating preview window
```

Comment:
Please mention about the default `align` value. I guess it is `item`.

Reply:
after I finish the title related, the doc will be update. I will continue to improve this PR in the next few days.